whisper/stt/processing/__init__.py

```diff
@@ -91,7 +84,18 @@ def __call__(self, *args, **kwargs):
     alignment_model = {}  # Alignement model(s) will be loaded on the fly
 
 
-def warmup():
-    model.check_loaded()
-    
-MODEL = (model, alignment_model)
+# Load ASR model
+model_type = os.environ.get("MODEL", "medium")
+logger.info(
+    f"Loading Whisper model {model_type} ({'local' if os.path.exists(model_type) else 'remote'})..."
+)
+try:
+    model = LazyLoadedModel(model_type, device=device, num_threads=NUM_THREADS)
+    MODEL = (model, alignment_model)
+    if str(device).lower() != "cpu":
```

Comment:
This test is a bit shaky.
Please use USE_GPU.
Also add a comment to explain why the warmup is done at the start with GPU.